faf.spec.in

```diff
@@ -108,6 +165,20 @@ development.
 Summary: %{name}'s WebUI rewrite
 Requires: %{name} = %{faf_version}
 Requires: httpd
+%if %{with python3}
+Requires: python3-mod_wsgi
+Requires: python3-flask >= 0.10
+Requires: python3-flask-wtf
+Requires: python3-flask-rstpages
+Requires: python3-flask-sqlalchemy
+Requires: python3-flask-openid
+Requires: python3-openid-teams
+Requires: python3-jinja2 >= 2.7
```

Comment:
And here.

Reply:
@xsuchy @marusak Thanks for the review. I have updated the spec file according to the feedback and our conversation today morning.